api/nakadi-event-bus-api.yaml

```diff
@@ -90,7 +90,7 @@ info:
     parameters (number of partitions, retention times, etc), as well as their modification.
 
 
-  version: '0.5.1'
+  version: '0.6.0'
```

Comment:
Bump the minor revision number because the object definitions have changed, but the API JSON is the same.